sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -771,6 +773,62 @@ def fill_matrix(*args):
               requires_partition_by=partitionings.Singleton(),
               proxy=proxy))
 
+  @frame_base.args_to_kwargs(pd.DataFrame)
+  @frame_base.populate_defaults(pd.DataFrame)
+  def corrwith(self, other, axis, **kwargs):
+    if axis not in (0, 'index'):
+      raise NotImplementedError('corrwith(axis=%r)' % axis)
+    if not isinstance(other, frame_base.DeferredFrame):
+      other = frame_base.DeferredFrame.wrap(
+          expressions.ConstantExpression(other))
+
+    if isinstance(other, DeferredSeries):
+      proxy = self._expr.proxy().corrwith(other._expr.proxy())
+      self, other = self.align(other, axis=0, join='inner')
+      corrs = [self[col].corr(other, **kwargs) for col in proxy.index]
```

Comment:
It looks like `method` is actually the only `kwarg` that overlaps between corr and corrwith. We should probably pass that explicitly, and handle the remaining arg, `drop`, here.

Reply:
Resolved.